seatunnel-core/seatunnel-core-starter/src/test/java/org/apache/seatunnel/core/starter/utils/FileUtilsTest.java

```diff
@@ -50,6 +53,18 @@ public void getConfigPath() throws URISyntaxException {
                 "flink.batch.conf", FileUtils.getConfigPath(sparkCommandArgs).toString());
     }
 
+    @Test
+    @DisabledOnOs(OS.WINDOWS)
+    void testExpectedError() {
+        SeaTunnelRuntimeException exception =
+                Assertions.assertThrows(
+                        SeaTunnelRuntimeException.class,
+                        () -> FileUtils.checkConfigExist(Paths.get("/tmp/not/existed")));
+        Assertions.assertEquals(
+                "ErrorCode:[COMMON-22], ErrorDescription:[SeaTunnel read file '/tmp/not/existed' failed, because it not existed.]",
+                exception.getMessage());
+    }
+
```

Comment:
```suggestion
    @Test
    void testExpectedError() {
        String root = File.listRoots()[0].getPath();
        // Unix Path: /tmp/not/existed
        // Windows Path: C:\tmp\not\existed
        Path path = Paths.get(root, "tmp", "not", "existed");
        SeaTunnelRuntimeException exception =
            Assertions.assertThrows(
                SeaTunnelRuntimeException.class,
                () -> FileUtils.checkConfigExist(Paths.get("/tmp/not/existed")));
        Assertions.assertEquals(
            "ErrorCode:[COMMON-22], ErrorDescription:[SeaTunnel read file '" + path.toString() + "' failed, because it not existed.]",
            exception.getMessage());
    }

```

Reply:
Cool! Addressed.